stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,8 @@
     <string name="stream_ui_send_label">Send</string>
     <string name="stream_ui_cancel_label">Cancel</string>
     <string name="stream_ui_attachment_downloading_started">Downloading started</string>
+    <string name="stream_ui_title_thread_reply">Thread Reply</string>
```

Comment:
This is unused, and we use a hardcoded string in the sample app's layout instead

Reply:
Now it is used for your second comment